src/ghga_datasteward_kit/models.py

```diff
@@ -124,24 +129,38 @@ def serialize(self, output_path: Path):
         os.chmod(path=output_path, mode=0o400)
 
     @classmethod
-    def load(cls, input_path: Path, selected_alias: str):
+    def load(cls, input_path: Path, selected_alias: str, selected_bucket: str):
```

Comment:
The `selected_bucket` param should be renamed to `fallback_bucket` to match the config parameter.